mlir/test/Dialect/Tosa/invalid.mlir

```diff
@@ -717,10 +717,21 @@ func.func @test_transpose_conv2d_invalid_outshape(%arg0: tensor<1x32x32x8xf32>,
 
 // -----
 
```

Comment:
do we need a i32 type check test?

Reply:
Can you expand on what you mean by "i32 type check test"? Are yous saying we need a negative test for the case that the shift parameter is a tensor of i32s?

Reply:
Well the shift shall be used only when with i32 inputs but not sure we can check this here as we make it compulsory.

Reply:
Okay I see your point now. I've made this an optional argument and then added another test for i32 to check it parses the shift operand. Thanks!